crates/shared/src/sources.rs

```diff
@@ -45,6 +45,7 @@ pub fn defaults_for_chain(chain_id: u64) -> Result<Vec<BaselineSource>> {
             BaselineSource::SushiSwap,
             BaselineSource::BalancerV2,
         ],
+        5 => vec![BaselineSource::UniswapV2, BaselineSource::SushiSwap],
```

Comment:
> Balancer (not all contracts needed are deployed, code needs to be ajdusted to exclude pool types that don't exist in the current network)

I think there is already a command line argument to control this:
https://github.com/cowprotocol/services/blob/03a4bc51806dfd03d9ba46ed65bb3136908691b4/crates/shared/src/arguments.rs#L129-L133

We would have to implement a `BalancerFactoryKind::defaults_for_chain` method and use that instead of `BalancerFactoryKind::value_variants`.

Reply:
Will implement this in a follow-up PR!

Reply:
🎉